### DISCUSSION TITLE
关于我在中国知网下载硕博论文pdf格式文件这件事

### DISCUSSION THREAD

**Discussion by giscus-xzonn** [Xzonn/xzonn.github.io#12] [General]
# 关于我在中国知网下载硕博论文pdf格式文件这件事

关于我在中国知网下载硕博论文pdf格式文件这件事 - 来自Xzonn的小站

https://xzonn.top/posts/Download-Pdf-File-from-Cnki.html

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****林士龙 评论于 2022-11-11 09:34:31 (UTC+8)*****

你好，我是通过tampermonkey找过来的，一直在使用“中国知网（CNKI）硕博论文跳转至“海外版”下载 pdf 文件”，但是下载后的论文都是加密的，无法在上面做笔记，所以想问一下能否提供密码，万分感谢
